### PR TITLE
Refactoring for subaccounts

### PR DESCRIPTION
**Pull request by davidyuk**

### CODE REVIEW

**Review by Liubov-crypto — COMMENTED** [review]
Web wallet: 
Login into the wallet - pass
Generate a new wallet - pass
Balances - pass
Search tokens - pass
Activity History - pass
Sort by Date - pass
Copy wallet address - pass
Change currency - pass
Change tokens - pass
Send Tokens - pass, using QR - pass
Send Tip - pass
Retip - pass
Register a name - pass
Generate invite link - pass
Claim back invite link - failed 
Top Up invite link - pass
Settings: 
Change language - pass
Security&Privacy - pass
Change Network - pass
Permission Settings - pass
Remove account - pass

FF/Chrome Extension: 
Generate new wallet - pass  
Login into the wallet - pass  
Change tokens - pass  
Change currency - pass  
Copy address - pass  
Send Tip - pass  
Send Tip to HTTP - pass (receive a warning)  
Tip via Twitter/Youtube button - pass 
Retip - pass  
Top Up ae tokens - pass  
Top Up FT- pass  
Connection with Jelly Swap -pass
 Withdraw (Send) - pass, via Scan QR code - pass  
Activity - pass, sort by date - pass
Tokens: search - pass 
 Send - pass  
Receive - pass  
Names: 
Set name pointer - pass 
 Extend - pass 
 Set as default name - pass  
Register name - pass (in the testnet)
Invite link:
Create - pass 
 Top up - pass  
Claim back - failed 
Settings:
 Language - pass 
 Security&Privacy - pass 
 Networks - pass 
 Permission Settings - pass  
Remove account - pass 

Got this error when claiming back tokens from the invitation link:
cl back inv link ff
errr1
errr2

Flying bug when switching between networks:
errro when switching networks

Well-known design bugs also remain #889 #849 #847 #802 and white stripes in FF browser/extension.

File: src/popup/utils/helper.js [hunk 1]
```diff
@@ -254,8 +166,6 @@ export const setBalanceLocalStorage = (balance) => {
 export const getBalanceLocalStorage = () =>
   localStorage.rxjs ? JSON.parse(localStorage.rxjs).balance : '0';
 
-export const getAeppUrl = (v) => new URL(v.connection.port.sender.url);
```
Comment (mradkov) [inline src/popup/utils/helper.js:257]: Isn't this used anywhere?

File: src/popup/utils/helper.js [hunk 2]
```diff
@@ -23,12 +23,8 @@ export const convertToken = (balance, precision) => BigNumber(balance).shiftedBy
 export const IN_FRAME = window.parent !== window;
 export const IN_POPUP = !!window.opener && window.name.startsWith('popup-');
 
-export const convertToAE = (balance) => +(balance / 10 ** 18).toFixed(7);
```
Comment (mradkov) [inline src/popup/utils/helper.js:26]: Isn't this used?

File: src/popup/utils/helper.js (hunk 1 above)
Comment (davidyuk) [reply src/popup/utils/helper.js:257]: it is not used, as far CI successfully builds the app

File: src/popup/utils/helper.js (hunk 2 above)
Comment (davidyuk) [reply src/popup/utils/helper.js:26]: yep

**Review by mradkov — APPROVED** [review]

**Pull request merged**